src/components/select/examples/select-multiple.tsx

```diff
@@ -38,21 +35,27 @@ export class SelectMultipleExample {
                     value={this.value}
                     options={this.options}
                     disabled={this.disabled}
+                    readonly={this.readonly}
                     required={this.required}
-                    onChange={this.onChange}
+                    onChange={this.changeHandler}
```

Comment:
dont we need to bind this function?

Reply:
No, because it's an arrow-function. `this` can't change in arrow-functions.